src/ng/compile.js

```diff
@@ -2592,11 +2587,13 @@ function $CompileProvider($provide, $$sanitizeUriProvider) {
             break;
 
           case '=':
-            if (optional && !attrs[attrName]) {
-              return;
+            if (!hasOwnProperty.call(attrs, attrName)) {
```

Comment:
@petebacondarwin - Re: ... see if this PR works for the latest on Material's master branch.  Is this something I should be doing now; or is someone else assigned this task ?